homeassistant/components/satel_integra/alarm_control_panel.py

```diff
@@ -122,24 +131,24 @@ def state(self):
 
         _LOGGER.debug("Disarming, self._state: %s", self._state)
 
-        await self.hass.data[DATA_SATEL].disarm(code)
+        await self._satel.disarm(code, [self._partition_id])
 
         if clear_alarm_necessary:
             # Wait 1s before clearing the alarm
             await asyncio.sleep(1)
-            await self.hass.data[DATA_SATEL].clear_alarm(code)
+            await self._satel.clear_alarm(code, [self._partition_id])
 
     async def async_alarm_arm_away(self, code=None):
         """Send arm away command."""
         _LOGGER.debug("Arming away")
 
         if code:
-            await self.hass.data[DATA_SATEL].arm(code)
+            await self._satel.arm(code, [self._partition_id])
 
     async def async_alarm_arm_home(self, code=None):
         """Send arm home command."""
         _LOGGER.debug("Arming home")
 
         if code:
-            await self.hass.data[DATA_SATEL].arm(
-                code, self._arm_home_mode)
+            await self._satel\
```

Comment:
Please break the line after the first parenthesis. That's the best way of formatting line breaks.
```py
await self._satel.arm(
    code, [self._partition_id], self._arm_home_mode)
```

Reply:
Fixed.